test/main.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+var assert = require('assert');
+var sinon = require('sinon');
+global.window = {
+  location:{
```

Comment:
Can you double-check this whole file for code-style violations? We use single-quotes, tabs, space after keywords, ...